apm-agent-attach/resources/NOTICE

```diff
@@ -178,15 +186,15 @@
    APPENDIX: How to apply the Apache License to your work.
 
       To apply the Apache License to your work, attach the following
-      boilerplate notice, with the fields enclosed by brackets "[]"
+      boilerplate notice, with the fields enclosed by brackets "{}"
       replaced with your own identifying information. (Don't include
       the brackets!)  The text should be enclosed in the appropriate
       comment syntax for the file format. We also recommend that a
       file or class name and description of purpose be included on the
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2018 - 2019 Elasticsearch BV
+   Copyright {yyyy} {name of copyright owner}
```

Comment:
🤔

Reply:
Copied from their repo 🤷‍♂

Reply:
I think this is GitHub misinterpreting the diff - some weird merge between LICENSE and NOTICE